### ISSUE TITLE
Add media doesn't work on mobile

### ISSUE THREAD

**Issue by stevejburge** [upstreamplugin/UpStream#178]
I don't think our approach to adding media works on mobile phones

- Add a task
- Click "Add media" in the description area

![screen shot 2017-09-25 at 7 14 00 am](https://user-images.githubusercontent.com/3868028/30805814-26699d36-a1c1-11e7-8a3c-c43cf5d4def0.png)

- The Media Library window will be too large, and it looks like the screen becomes completely white
- You may also swipe left and see a lot of empty screen space
- Pinching also may reveal unnecessary empty space on the screen

Workaround: Add this CSS:

```
.dataTables_wrapper {
overflow: auto;
}

```

**Comment by upstreamplugin** [comment]
A follow-on suggestion from the same customer ...

- Click Add Media
- Click Close Media Panel
- Scroll all the way up (keep scrolling up). You will inadvertently refresh the page

Workaround is to add this CSS:
```
.nav-md .modal {
overflow: auto !important;
}

.nav-sm .modal {
overflow: auto !important;
}

```
Add this to the end of upstream-frontend-edit.js:

```
jQuery( '.nav-md .modal' ).scroll(
function() {
jQuery( 'body' ).addClass( 'modal-open' );
}
);
jQuery( '.nav-sm .modal' ).scroll(
function() {
jQuery( 'body' ).addClass( 'modal-open' );
}
);
```

**Comment by awijasa** [comment]
Hello,

Just want to add that on mobile, Media Library window becomes too large only when there is a title that is too long: #177

Thanks,

Adrian

**Comment by deenison** [comment]
@awijasa is right.

This will be patched on `v1.12.2` along side #177 since it's a collateral problem.
I'm closing this issue marking it as `worksforme`.

Thank you all.

**Comment by upstreamplugin** [comment]
-- Please reply above this line --


Hello Upstreamplugin/upstream,

We received your email request! 

Our support hours are 9am - 5pm EST, Monday to Friday. We'll try
to respond to all tickets within 24 hours, Monday to Friday.

Thank you!

Steve and the UpStream team
